types/brotli/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for brotli 1.3
+// Project: https://github.com/foliojs/brotli.js
+// Definitions by: Artur Androsovych <https://github.com/arturovt>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+import compress = require('./compress');
+import decompress = require('./decompress');
+
+export { compress, decompress };
```

Comment:
If you want, you can follow this TS pattern for exports:
```suggestion
export import compress = require('./compress');
export import decompress = require('./decompress');
```